sky/cli.py

```diff
@@ -114,6 +114,18 @@ def _get_glob_clusters(clusters: List[str]) -> List[str]:
     return list(set(glob_clusters))
 
 
+def _get_glob_storages(storages: List[str]) -> List[str]:
+    """Returns a list of storages that match the glob pattern."""
+    glob_storages = []
+    for storage_object in storages:
+        glob_storage = global_user_state.get_handle_from_storage_name(
```

Comment:
If I run `sky storage delete *`, I get the error: 
```
(base) romilb@romilbx1yoga:~/tmp$ sky storage ls
No existing storage.
(base) romilb@romilbx1yoga:~/tmp$ sky storage delete *
Storage gello.txt not found.
```

There is likely something going wrong with the SQL query since gello.txt is a file that was uploaded, and was never used a storage name.

Reply:
The quotation marks are needed around "*" are needed, otherwise this will fetch the files in whichever current directory you are running from and not the storage object names.

Reply:
Ah my bad - thanks for pointing this out!